pkg/server/plugin/upstreamauthority/vault/vault_client.go

```diff
@@ -115,6 +126,8 @@ func NewClientConfig(cp *ClientParams, logger hclog.Logger) (*ClientConfig, erro
 	defaultParams := &ClientParams{
 		CertAuthMountPoint:    defaultCertMountPoint,
 		AppRoleAuthMountPoint: defaultAppRoleMountPoint,
+		K8sAuthMountPoint:     defaultK8sMountPoint,
+		K8sAuthTokenPath:      defaultK8sTokenPath,
```

Comment:
Please look at the comment about assuming a default token path.

Reply:
fa226e1